src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -324,45 +325,51 @@ def _validate_service_principal(client, sp_id):
 
 
 def _build_service_principal(client, name, url, client_secret):
-    sys.stdout.write('creating service principal')
+    view = progress.DeterminateStandardOut()
```

Comment:
@troydai, I just don't see any gain by doing that, because it doesn't make code any cleaner.
Can you elaborate me the reason? If it is just the policy, i can definitely change.